### ISSUE TITLE
Inappropriate use of `aria-hidden="true"` attribute on Instagram link in footer

### ISSUE THREAD

**Issue by Christopher-Green424** [ToriLemons/QA-Practice-One#18]
## Devices
Device name:	[Custom Desktop PC](https://pcpartpicker.com/list/JxkYvf "PCPartPicker build")
Processor:	Intel Core i5-6600K 3.5 GHz Quad-Core Processor
System type:	Windows 10 Pro 64-bit operating system, x64-based processor
Resolution:	1920x1080 pixels, 16:9 ratio (~92 PPI density)

Brave Browser Version Version 1.29.79 Chromium: 93.0.4577.63 (Official Build) (64-bit)

## Expected Behavior
Screenreader should be able to read all functional parts of a website, such as links in the navbar or footer.

## Actual Behavior
There's an aria-hidden="true" attribute in the Instagram social media links on both the homepage and contact page. This makes screenreaders skip past it, though Windows Narrator seems to be able to focus on it anyways.

## Screenshots
![Screenshot 2021-10-04 132454](https://user-images.githubusercontent.com/82982114/135919719-83d0e25d-78b7-4876-ba4c-479be66bbcb3.png)
